Sources/LogtoSocialPluginWechat/LogtoSocialPluginWechat.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  LogtoSocialPluginWechat.swift
+//
+//
+//  Created by Gao Sun on 2022/4/1.
+//
+
+import Foundation
+import LogtoSocialPlugin
+@_exported import WechatOpenSDK
+
+public enum LogtoSocialPluginWechatError: LogtoSocialPluginError {
+    case authFailed(withCode: String)
+    case insufficientInfo
+    case noCodeReturned
+
+    public var code: String {
+        switch self {
+        case .insufficientInfo:
+            return "insufficient_info"
+        case .noCodeReturned:
+            return "no_code_return"
+        case .authFailed:
+            return "auth_failed"
+        }
+    }
+
+    var localizedDescription: String {
+        switch self {
+        case .insufficientInfo:
+            return "The redirect URL contains insufficient information."
+        case .noCodeReturned:
+            return "Wechat didn't return authorization `code`."
+        case let .authFailed(withCode):
+            return "Alipay auth failed with code \(withCode)."
+        }
```

Comment:
Is this `authFailed(withCode)` specific to Alipay?

Reply:
sry i copied from alipay, should be wechat

Reply:
sry i copied from alipay, should be wechat

Reply:
sry i copied from alipay, should be wechat

Reply:
ok